packages/core/src/types/VideoViewportProperties.ts

```diff
@@ -0,0 +1,17 @@
+import { ViewportProperties } from './ViewportProperties';
+import Point2 from './Point2';
+
+/**
+ * Stack Viewport Properties
+ */
+type VideoViewportProperties = ViewportProperties & {
+  loop?: boolean;
+  muted?: boolean;
```

Comment:
hmmmm, since we are using offscreen video tag, do we really support audio? I'm confused